pkgs/unified_analytics/example/serving_surveys.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:clock/clock.dart';
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+
+import 'package:unified_analytics/src/constants.dart';
+import 'package:unified_analytics/src/enums.dart';
+import 'package:unified_analytics/src/survey_handler.dart';
+import 'package:unified_analytics/unified_analytics.dart';
+
+void main() async {
+  late final MemoryFileSystem fs;
+  late final Analytics analytics;
+  late final Directory home;
```

Comment:
@DanTup this example file should make it easier for us to discuss any changes to the workflow (if any are needed)

Reply:
@bwilkerson also an fyi if this example file workflow...  from the analysis server side, does this workflow make sense to you?